contracts/burner/src/msg.rs

```diff
@@ -14,6 +14,6 @@ pub struct MigrateMsg {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct InitMsg {}
 
-/// HandleMsg is a placeholder where we don't take any input
+/// ExecuteMsg is a placeholder where we don't take any input
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
-pub struct HandleMsg {}
+pub struct ExecuteMsg {}
```

Comment:
By the way, it would be very nice to make the `execute` entry point optional as well. It is default to have it, but we have at least 2 example contracts where the entry point is not used. But this is a different PR and not urgent.

Reply:
I'll create an issue for this.